src/lib/exhaust.ts

```diff
@@ -0,0 +1,31 @@
+import {ExhaustExportCsv} from '../models/exhaust-export-csv';
+
+const createExhaustPlugin = (pluginTypeName: string) => {
+  switch (pluginTypeName) {
```

Comment:
we should discuss if plugins are another repository or builtins

Reply:
I think we got approval to import exhaust plugins as we are doing with ordinary plugins

Reply:
in retrospect i think this makes sense.
looking at the way ordinary plugins are being created from the manifest, i think applying a similar flow for creation of the exhaust plugins means significant changes. i still think we should do it :) but maybe not in the scope of this task.
i propose maybe pushing the csv-export exhaust functionality as it is in this PR, and create a new issue (with the relevant priority) for refactoring it into external plugins. WDYT?